redux/actions/conversations.js

```diff
@@ -7,6 +7,7 @@ export const FETCHING_CONVERSATIONS = 'FETCHING_CONVERSATIONS';
 export const fetchMyConversations = () => async (dispatch, getState) => {
```

Comment:
This file is small for now, you should consider splitting up your types and actions in order to accommodate for scalability.